packages/inter-protocol/test/psm/test-psm.js

```diff
@@ -182,7 +182,10 @@ async function makePsmDriver(t, customTerms) {
     harden({
       feeMintAccess,
       initialPoserInvitation,
-      storageNode: mockChainStorage.makeChildNode('thisPsm'),
+      storageNode: mockChainStorage
+        .makeChildNode('psm')
+        .makeChildNode('IST')
+        .makeChildNode('AUSD'),
```

Comment:
this looks like logic that is meant to be under test but is part of the driver.

consider a function to make a storage path for a PSM which can be re-used in startPSM:
https://github.com/Agoric/agoric-sdk/blob/50c8e2681fe8bd668278fde3e97c0333e8803cfd/packages/inter-protocol/src/proposals/startPSM.js#L129-L133